packages/components/src/Components/ErrorState/ErrorState.styles.scss

```diff
@@ -0,0 +1,3 @@
+@import "~@patternfly/patternfly/patternfly.css";
```

Comment:
No need to include these, they are already included in chrome. Looking at these styles, they can be safely removed.